### PR TITLE
Chapter 9 - Advanced Login

### PR DESCRIPTION
**Pull request by EvonneNgXinYi**
Remember user

### CODE REVIEW

**Comment by EvonneNgXinYi** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -3,15 +3,36 @@ def log_in user
...
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
   def current_user
-    @current_user ||= User.find_by id: session[:user_id] if session[:user_id]
+    if session[:user_id]
+      @current_user ||= User.find_by id: session[:user_id]
+    elsif user_id = cookies.signed[:user_id]
+    user = User.find_by id: user_id
```
Comment (chungpv-1008) [inline app/helpers/sessions_helper.rb]:
```suggestion
      user = User.find_by id: user_id
```

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -3,15 +3,36 @@ def log_in user
...
+    user.forget
+    cookies.delete :user_id
+    cookies.delete :remember_token
   end
 
   def logged_in?
     current_user.present?
   end
-
+
```
Comment (chungpv-1008) [inline app/helpers/sessions_helper.rb]:
```suggestion

```

**Review by chungpv-1008 — COMMENTED** [review]
please fix comment, then merge

**Pull request merged**